Build failure with Cabal 2.0

```
[4 of 4] Compiling Main             ( Setup.hs, Setup.o )

Setup.hs:29:34: error:
    Ambiguous occurrence ‘Version’
    It could refer to either ‘Distribution.Simple.Version’,
                             imported from ‘Distribution.Simple’ at Setup.hs:11:1-26
                             (and originally defined in ‘Distribution.Version’)
                          or ‘Data.Version.Version’,
                             imported from ‘Data.Version’ at Setup.hs:16:1-19
   |
29 | data Aux = Aux { guideVersion :: Version }
   |                                  ^^^^^^^
```

--- a/Setup.hs
+++ b/Setup.hs
@@ -10,7 +10,7 @@ import Control.Applicative ((<$>))
 -- Cabal
 import Distribution.Simple
 import Distribution.PackageDescription
-import Distribution.PackageDescription.Parse (readPackageDescription)
+import Distribution.PackageDescription.Parsec (readGenericPackageDescription)
 import Distribution.Verbosity (normal)
 -- Version
 import Data.Version
@@ -30,7 +30,7 @@ data Aux = Aux { guideVersion :: Version }
 
 getAux :: IO Aux
 getAux = do
-  pd <- packageDescription <$> readPackageDescription normal "hatex-guide.cabal"
+  pd <- packageDescription <$> readGenericPackageDescription normal "hatex-guide.cabal"
   return $ Aux (pkgVersion $ package pd)
 
 auxmodule :: Aux -> String

--- a/Setup.hs
+++ b/Setup.hs
@@ -12,8 +12,6 @@ import Distribution.Simple
 import Distribution.PackageDescription
 import Distribution.PackageDescription.Parsec (readGenericPackageDescription)
 import Distribution.Verbosity (normal)
--- Version
-import Data.Version
 
 main :: IO ()
 main = do
@@ -45,5 +43,5 @@ auxmodule a = unlines [
  , "-- | The version of the guide. Based on the version of the package."
  , "guideVersion :: Version"
  , "guideVersion = " ++ (let v = guideVersion a
-                         in  "Version " ++ show (versionBranch v) ++ " []")
+                         in  "makeVersion " ++ show (versionNumbers v))
    ]

